### DISCUSSION TITLE
Frame Rate

### DISCUSSION THREAD

**Discussion by p1pster** [emgucv/emgucv#434] [Q&A] [answered]
Hi all, I am tryign to set a specific FPS for the capture  with the following:

`_capture = new VideoCapture((Int32)form.selected_value);
_capture.SetCaptureProperty(CapProp.Fps, 10);`

However it makes no difference to the actual capture and still runnign at 30fps. Is this the right property to be changing to adjust this, or do i need to manual capture at the desired frame rate?

I have looked throught the documentation but cannot find any further propreties which need to be enabled.

**Comment by emgucv** [comment, marked as answer]
Check the return value of _capture.SetCaptureProperty, if it is false, setting the specific property on your current backend is not supported.

https://www.emgu.com/wiki/files/4.5.1/document/html/562586c1-1319-fdb3-9121-ed2f81a53f9d.htm

> Return Value
> Type: Boolean
> True if success

https://docs.opencv.org/4.5.1/d8/dfe/classcv_1_1VideoCapture.html#a8c6d8c2d37505b5ca61ffd4bb54e9a7c

> Note
> Even if it returns true this doesn't ensure that the property value has been accepted by the capture device.

**Reply by p1pster** [reply]
Hi thanks for this reply, I have checked the return value of this and it returns true, however, when i look at the properties of an Imagebox it reports that it runs between 47-50 fps. Im think its just the camera quality, it should run at the resoltion im capturing at at least 60fps.
